src/joystick/SDL_gamepad_db.h

```diff
@@ -29,6 +29,9 @@
    Alternatively, you can use the app located in test/controllermap
  */
 static const char *s_GamepadMappings[] = {
+#ifdef SDL_JOYSTICK_PRIVATE
+    SDL_PRIVATE_GAMEPAD_DEFINITIONS()
```

Comment:
This probably shouldn't be a function macro. Just something like:
```c
#define SDL_PRIVATE_GAMEPAD_DEFINITIONS \
 "xinput,*,...", \
 "sparkplug,*,..."
 ```

Reply:
On the private side that's exactly what it is, I just thought having the parenths would make it clearer that some preprocessor shenanigans were going on here. Same thought process for the PromptAssertion macro. I'm totally fine with changing this to just a regular define.

Reply:
No, parentheses for static initialization data doesn't make any sense. It's fine over in the assertion code, but here you should just have a `#define`, no function macro.